### PR TITLE
Fix for iOS Application Installed event

### PR DESCRIPTION
**Pull request by b099l3**
### Context
This PR is a workaround fix for the `Application Installed` events not firing when setting the config via dart and not via a file (`info.plist`), issue #26.

### Fix Details:
Due to the method sequence and the way the native library works, [described here](https://github.com/la-haus/flutter-segment/issues/26#issuecomment-1215282446), we need to either call the dart code `Segment.config()` in `didFinishLaunchingWithOptions()`, similar to how the configFromFile works **or** send the `Application Installed` event when we call the dart code `Segment.config()`. 

As I dont think there is an easy way to call `Segment.config()` in the native code `didFinishLaunchingWithOptions()`, I have instead opted to add a similar method to the way the native code works when sending `Application Installed` / `Application Updated` events, this code is [here](https://github.com/segmentio/analytics-ios/blob/fc64997865619f73bbab196c164f7845a13da110/Segment/Classes/SEGAnalytics.m#L163).

This will make use of the same `NSUserDefaults` keys as the native library and manually send the events.

This maybe isn't the best way to explain this fix so happy to chat. Also keep up the good work 💙.

### CODE REVIEW

**Comment by b099l3** [comment]
I have tested this and can see the events being fired 
image

File: example/ios/Runner/Info.plist [hunk 1]
```diff
@@ -39,10 +39,6 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
-	<key>com.claimsforce.segment.WRITE_KEY</key>
-	<string>YOUR_WRITE_KEY_GOES_HERE</string>
-	<key>com.claimsforce.segment.TRACK_APPLICATION_LIFECYCLE_EVENTS</key>
-	<false/>
```
Comment (danielgomezrico) [inline example/ios/Runner/Info.plist:45]: Is this related to this PR?

**Review by danielgomezrico — APPROVED** [review]
Thanks for this PR, looks good to me
